docs/docsite/rst/inventory/implicit_localhost.rst

```diff
@@ -26,6 +26,7 @@ You can override the built-in implicit version by creating a ``localhost`` host
 
 .. note::
   - This host is not targetable via any group, however it will use vars from ``host_vars`` and from the 'all' group.
+  - The `inventory_file` and `inventory_dir` magic variables are not available for the implicit localhost as they are dependent on **each inventory host**.
```

Comment:
Sphinx uses double backticks for `code/command text` - please update to ``inventory_file`` and ``inventory_dir`` on this line.